packages/graphile-build-pg/src/plugins/PgTypesPlugin.ts

```diff
@@ -874,7 +881,7 @@ export default (function PgTypesPlugin(
               fields: {
                 value: {
                   description: "The value at one end of our range.",
-                  type: new GraphQLNonNull(gqlRangeSubType),
+                  type: gqlRangeInputSubType,
```

Comment:
I don't understand how this was working before, since there was a mismatch between input types and output types. I think that the Typescript definitions are too loose around this.